tests/test_sweep_integration.py

```
@@ -0,0 +1,435 @@
+"""
+Integration tests for sweep functionality using real data.
+
+This module contains integration tests that use real SLEAP experiment data
+to test the sweep functionality. It includes both mocked tests for isolated
+testing and pure integration tests that test the actual workflow.
+"""
+
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)